tests/docker/instance_activate_ipsec_lb_agent_resp

```diff
@@ -28,7 +28,12 @@
                                 "Type": "tcp",
                                 "PrivatePort": 8080
                             }
-                        ]
+                        ],
+                        "Labels": {
```

Comment:
Not an explicit reason. I figured it would be better to have that type information if we needed it and couldn't see any down side to it versus true|false.